### PR TITLE
distinguish class names from binary names

### PR DESCRIPTION
**Pull request by elharo**
@suztomo

### CODE REVIEW

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java [hunk 1]
```diff
@@ -28,29 +28,35 @@
...
+   * found in the .class files.
+   */
+  public String getBinaryName() {
+    return binaryName;
   }
 
   /**
-   * Returns {@link ClassFile} with top level class if this class is an inner class by checking "$"
-   * character; otherwise the instance itself.
+   * Returns the outer class class if this class is an inner class.
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]:
```suggestion
   * Returns the outer class if this class is an inner class.
```

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java [hunk 2]
```diff
@@ -28,29 +28,35 @@
...
     return jar;
   }
 
-  /** Returns class name (binary name as in {@link Symbol#getClassName()}) */
-  public String getClassName() {
-    return className;
+  /**
+   * Returns the binary name of the class as defined by the 
+   * <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.1">Java
+   * Language Specification</a> and
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: Unnecessary new line.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java [hunk 3]
```diff
@@ -28,29 +28,35 @@
...
+   * found in the .class files.
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]:
```suggestion
   * found in the .class file.
```

Or this doesn't need "and found in the .class file" part.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java [hunk 4]
```diff
@@ -17,7 +17,7 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-
+import com.google.common.annotations.VisibleForTesting;
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: Add an empty line between static import and non-static import.

Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: done

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java (hunk 2 above)
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: done

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java (hunk 3 above)
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: done

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java (hunk 1 above)
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java]: done

**Pull request merged**